### DISCUSSION TITLE
when i add email address for the github it shows already  in use

### DISCUSSION THREAD

**Discussion by Nipunmihi** [community/community#110898] [GitHub Education]
### Select Topic Area

Question

### Body

Dear commiunity my university already buy student devlop pack from the github and we can use adding university email .but when i add my univercity email address it shows email address already use.pleasse kindly check the my request.i have to assigment using the github

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Nipunmihi , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!


Please note: I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Reply by nigelandamCvSU** [reply]
Hello, any feedback from this one?

**Comment by Sitevity** [comment]
I seems You have Already Applied for this
